app/views/admin/pages/show.html.erb

```diff
@@ -1,11 +1,11 @@
 <form id="slices-form">
   <div id="container-actions">
-    <% if Slices::Translations.available? %>
+    <% if I18n.available_locales.many? %>
       <div class="toolbar translations">
         <nav>
           Edit in
           <ul>
-            <% Slices::Translations.all.each do |locale, language| %>
+            <% I18n.available_locales.each do |locale, language| %>
```

Comment:
I was a bit hasty here - `I18n.available_locales` does not give you the `language`, just the `locale`! I'll see if there's an easy way to get the former from the latter.